store/confirmCDS.go

```diff
@@ -0,0 +1,166 @@
+package store
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/bitmark-inc/autonomy-api/schema"
+)
+
+type CDSCountryType string
+
+const (
+	CdsUSA         = "United States"
+	CdsUSALevel    = "county"
```

Comment:
this variable is not used.

Reply:
It is used when I do CDSUS CDSTaiwan, but since I take out CDSTaiwan, it is no used now. I think it may be used in the future.